src/migrations/20220203152515-bootstrap_accounts.ts

```diff
@@ -0,0 +1,95 @@
+/* eslint @typescript-eslint/no-var-requires: "off" */
+// @ts-expect-error ts complain about "Cannot redeclare block-scoped variable 'crypto'"
+const crypto = require("crypto")
+
+const adminUsers = [
```

Comment:
re-import because migrate mongo use export.default, not sure how to switch to import/export.

maybe you have some clue @samerbuna @jotapea ?